Minor fixs in Makefiles clean rules
 rm autogenerated simavr/sim_core_conf.h & sim_core_decl.h
 rm *.hex in examples/boards_*/
 .
 simavr (1.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #568156)
Author: Laurent Navet (Mali) <laurent.navet@gmail.com>
Bug-Debian: http://bugs.debian.org/568156

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- simavr-1.0.orig/examples/board_ledramp/Makefile
+++ simavr-1.0/examples/board_ledramp/Makefile
@@ -46,4 +46,4 @@ ${target}: ${board}
 	@echo $@ done
 
 clean: clean-${OBJ}
-	rm -rf *.a *.axf ${target} *.vcd
+	rm -rf *.a *.axf ${target} *.vcd *.hex
--- simavr-1.0.orig/examples/board_simduino/Makefile
+++ simavr-1.0/examples/board_simduino/Makefile
@@ -47,4 +47,4 @@ ${target}: ${board}
 	@echo $@ done
 
 clean: clean-${OBJ}
-	rm -rf *.a *.axf ${target} *.vcd
+	rm -rf *.a *.axf ${target} *.vcd *.hex
--- simavr-1.0.orig/examples/board_timer_64led/Makefile
+++ simavr-1.0/examples/board_timer_64led/Makefile
@@ -47,4 +47,4 @@ ${target}: ${board}
 	@echo $@ done
 
 clean: clean-${OBJ}
-	rm -rf *.a *.axf ${target} *.vcd
+	rm -rf *.a *.axf ${target} *.vcd *.hex
--- simavr-1.0.orig/simavr/Makefile
+++ simavr-1.0/simavr/Makefile
@@ -83,6 +83,7 @@ ${target}	: ${OBJ}/${target}.elf
  
 clean: clean-${OBJ}
 	rm -rf ${target} *.a *.so
+	rm -f sim_core_*.h
 
 DESTDIR = /usr/local
 PREFIX = ${DESTDIR}
